tests/components/config/test_area_registry.py

```diff
@@ -9,15 +13,20 @@
 from tests.common import ANY
 
 
-@pytest.fixture
-async def client(hass, hass_ws_client):
+@pytest.fixture(name="client")
+async def client_fixture(
+    hass: HomeAssistant,
+    hass_ws_client: Callable[
```

Comment:
This is typed with this at the source:
https://github.com/home-assistant/core/blob/9b1ba19354f43af75bc58d4c62beb2c3129aa1e3/tests/typing.py#L33

Reply:
Yeah just noticed we have an typing alias for it. Adjusting it.